src/blog/2024/05/node-red-mind-stack-with-flowfuse.md

```diff
@@ -0,0 +1,207 @@
+---
+title: The MIND stack with Node-RED and FlowFuse Dashboard 2.0
+subtitle: Our objective is to streamline the deployment of the MIND stack, enhancing its usability without compromising functionality.
+description: Our objective is to streamline the deployment of the MIND stack, enhancing its usability without compromising functionality.
+date: 2024-05-06
+authors: ["harshad-joshi","grey-dziuba"]
+image: /blog/2024/05/images/mind-mqtt-influxdb-node-red-dashboard-2-0.png
+tags:
+    - posts
+    - flowfuse
+    - mind
+    - ming
+    - influx
+    - dashboards
+---
+
+The MIND stack, which includes MQTT, InfluxDB, Node-RED, and Grafana is a popular choice for IoT and IIoT-based applications, but in this article, we are going to leverage Dashboards 2.0 for our visualizations, and instead of MING we are going to call it the MIND stack (MQTT, INfluxDB, Node-RED and Node-RED Dashboard 2.0 by FlowFuse). To demonstrate this, we will design a dashboard to display data collected from a Temperature Sensor that sends Relative Humidity (RH) and Temperature values to an MQTT broker. This application allows users to visualize historical data in Tabular form and view it live using various Dashboard 2.0 objects.
```

Comment:
```suggestion
The [MING stack](/blog/2023/02/ming-blog/), which includes MQTT, InfluxDB, Node-RED, and Grafana is a popular choice for IoT and IIoT-based applications, but in this article, we are going to leverage Dashboards 2.0 for our visualizations, and instead of MING we are going to call it the MIND stack (MQTT, INfluxDB, Node-RED and Node-RED Dashboard 2.0 by FlowFuse). To demonstrate this, we will design a dashboard to display data collected from a Temperature Sensor that sends Relative Humidity (RH) and Temperature values to an MQTT broker. This application allows users to visualize historical data in Tabular form and view it live using various Dashboard 2.0 objects.
```